.github/dependabot.yml

```diff
@@ -12,10 +12,17 @@ updates:
         update-types:
           - "version-update:semver-minor"
           - "version-update:semver-patch"
+    cooldown:
+      # https://blog.yossarian.net/2025/11/21/We-should-all-be-using-dependency-cooldowns
+      # Cooldowns protect against supply chain attacks by avoiding the
+      # highest-risk window immediately after new releases.
+      default-days: 14
```

Comment:
I also included these cooldowns from https://github.com/python/cpython/pull/141866 to make zizmor happy.